massgov/mayflower
# Fix release automation month/day display in changelog.md file

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

The display for the month and day was not displaying correctly in the release automation. Update the script to fix that issue. 

The `getMonth()` uses 0 for month of January and 11 for December. I adjusted to add +1 to display it correctly. https://softwareengineering.stackexchange.com/questions/108495/why-getmonth-starts-with-0

The `getDay()` can return the day of the week for an integer so I adjusted it to use `getDate()` instead which works better. https://stackoverflow.com/questions/13359294/date-getday-javascript-returns-wrong-day

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-16732)
- [Github issue]()

## Steps to Test


1. Checkout the branch
2. Comment out anything below `const title = ## ${minor} (${month}/${day}/${year});`
3. Add `console.log(title);` to script
4. Run the following command `node scripts/release-branch.js` (see image below from output)

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

![Screen Shot 2020-01-08 at 9 52 15 PM](https://user-images.githubusercontent.com/19477691/72034829-1c3b7b00-3264-11ea-9636-b5db717b4b66.png)


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
